crates/db_schema/src/lib.rs

```diff
@@ -178,6 +182,7 @@ pub enum SearchType {
   Users,
 }
 
+// TODO I think this should go away in favor of FollowState
```

Comment:
You can see the difference between these two in the [conversion function](https://github.com/LemmyNet/lemmy/blob/main/crates/db_schema/src/impls/community.rs#L463). `Accepted` and `Subscribed` have different names but that doesnt matter. The main thing is that "not following" is represented by null (no row exists) in the database and in `CommunityFollowerState`, but `SubscribedType` represents it with an explicit value `NotSubscribed`. 

It looks like you already replaced most uses of `SubscribedType` with the db type `CommunityFollowerState`, only `PendingFollow.subscribed` is remaining. So you can also change that to `pub subscribed: Option<CommunityFollowerState>` and get rid of `SubscribedType`.

Reply:
K, figured it out I think.